package.json

```diff
@@ -4,7 +4,7 @@
   "author": "Tobias Koppers @sokra",
   "description": "Extract text from bundle into a file.",
   "peerDependencies": {
-    "webpack": "^2.1.0-beta.19 || ^2.2.0-rc.0"
+    "webpack": "^2.1.0-beta.19 || ^2.2.0"
   },
```

Comment:
`"webpack": "^2.1.0-beta.19 || ^2.2.0-rc.0 || ^2.2.0"`

You can't take out the rc. Just add 2.2.0 as another OR